app/src/main/java/org/matrix/matrixandroidsdk/services/EventStreamService.java

```diff
@@ -150,9 +150,11 @@ public void onBingEvent(Event event, RoomState roomState, BingRule bingRule) {
                 return;
             }
 
+            String roomName = room.getName(mSession.getMyUser().userId);
```

Comment:
Is there ever a chance of ``mSession.getMyUser()`` being ``null`` (e.g. because they logged out and the notification arrives after they logout)? Not sure if this can happen; just asking if you've thought of this.